src/Tracking/README.md

```diff
@@ -284,3 +284,54 @@ $this->conditionally_share_with_tags( Loaded::class );
 ```
 
  /Dev Info -->
+
+<woo-tracking-jsdoc>
+<!---
+Everything below will be automatically generated by `woo-tracking-jsdoc`.
+Do not edit it manually!
+-->
+
+### [`gla_ces_feedback`](js/src/components/customer-effort-score-prompt/index.js#L29)
+CES feedback recorded
+#### Emitters
+- [`CustomerEffortScorePrompt`](js/src/components/customer-effort-score-prompt/index.js#L49) whenever the CES feedback is recorded
+
+### [`gla_ces_modal_open`](js/src/components/customer-effort-score-prompt/index.js#L24)
+CES modal open
+#### Emitters
+- [`CustomerEffortScorePrompt`](js/src/components/customer-effort-score-prompt/index.js#L49) whenever the CES modal is open
+
+### [`gla_ces_snackbar_closed`](js/src/components/customer-effort-score-prompt/index.js#L19)
+CES prompt snackbar closed
+#### Emitters
+- [`CustomerEffortScorePrompt`](js/src/components/customer-effort-score-prompt/index.js#L49) whenever the CES snackbar (notice) is closed
+
+### [`gla_ces_snackbar_open`](js/src/components/customer-effort-score-prompt/index.js#L14)
+CES prompt snackbar open
+#### Emitters
+- [`CustomerEffortScorePrompt`](js/src/components/customer-effort-score-prompt/index.js#L49) whenever the CES snackbar (notice) is open
+
+### [`gla_edit_mc_store_address`](js/src/components/contact-information/store-address-card.js#L124)
+"Edit MC store address" Tracking event
+#### Properties
+|   |   |   |
+|---|---|---|
+`path` | `string` | A page from which the link was clicked.
+`subpath` | `string\|undefined` | A subpage from which the link was clicked.
```

Comment:
🚧 

In my MD reader, I the `|` breaks the table like:

<img width="332" alt="Screenshot 2022-04-06 at 10 50 30" src="https://user-images.githubusercontent.com/5908855/161917633-50766d9e-7ffb-46d9-8a5e-ad8e67343fa5.png">

After [some investigation](https://www.markdownguide.org/extended-syntax/#escaping-pipe-characters-in-tables) seems like the right way to do this is with HTML character code (&#124;). However, I tried it locally and didn't work inside the backticks, so I guess our best try is to do something like

 \`string\`\&# 124;\`undefined\`

Reply:
Seems like it's a problem with my IDE reader since I see Github is rendering properly the table. 

So this is now 💅  instead of  🚧  and could be ignored

Reply:
Thanks for the investigation. Now, I'm not sure which way to go. 
There is no official single spec for Markdown tables. The original one https://daringfireball.net/projects/markdown/syntax#backslash does not mention tables.
On one hand `foo \| bar` is easier to write and read while reading the code and making writing int the easiest is the goal of this PR. Also, the GitHub flavored MD specifies `\|` as an escape https://github.github.com/gfm/#example-200

On the other hand, this clearly doesn't make reading easy for you and other devs using your IDE. :confused:

Reply:
I don't really mind to be honest,  I can read directly in the code / md  and since now is working I guess its better to give Github more priority than IDE. So my suggestion is to keep it like you did 👌